siriuspy/siriuspy/devices/lillrf.py

```diff
@@ -40,25 +45,80 @@ def amplitude(self, value):
 
     @property
     def phase(self):
-        """."""
+        """Phase."""
         return self['GET_PHASE']
 
     @phase.setter
     def phase(self, value):
         self['SET_PHASE'] = value
 
+    @property
+    def integral_enable(self):
+        """Integral Enable."""
+        return self['GET_INTEGRAL_ENABLE']
+
+    @integral_enable.setter
+    def integral_enable(self, value):
+        self['SET_INTEGRAL_ENABLE'] = value
+
+    @property
+    def feedback_state(self):
+        """Feedback State."""
+        return self['GET_FB_MODE']
+
+    @feedback_state.setter
+    def feedback_state(self, value):
+        self['SET_FB_MODE'] = value
+
+    @property
+    def i_ref(self):
+        """I reference."""
+        return self['GET_CH1_SETTING_I']
+
+    @property
+    def q_ref(self):
+        """Q reference."""
+        return self['GET_CH1_SETTING_Q']
+
+    @property
+    def i_mon(self):
+        """I monitor."""
+        return self['GET_CH1_I']
+
+    @property
+    def q_mon(self):
+        """Q monitor."""
+        return self['GET_CH1_Q']
+
     def cmd_set_phase(self, value, timeout=10):
-        """."""
+        """Set and wait for phase property to reach value."""
         self.phase = value
         self._wait_rb_sp(timeout, 'phase')
 
     def cmd_set_amplitude(self, value, timeout=30):
-        """."""
+        """Set and wait for amplitude property to reach value."""
         self.amplitude = value
         self._wait_rb_sp(timeout, 'amplitude')
 
+    def cmd_set_integral_enable(self, value, timeout=3):
+        """Set and wait for integral enable property to reach value."""
+        self.integral_enable = value
+        self._wait('GET_INTEGRAL_ENABLE', value, timeout=timeout)
+
+    def cmd_set_feedback_state(self, value, timeout=3):
```

Comment:
@anacso17 , my ideia was too have cmds to turn on and turn off the feedback and the integral enable.

Reply:
oh, I see, together in the same method, just like injection system handlers?

Reply:
like the ones we have in other Devices.